release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3219,6 +3219,8 @@ Starting with {product-title} 4.8.34, support for using the Cloud Credential Ope
 
 * Previously, the `SystemMemoryExceedsReservation` alert using Prometheus QL was using `hugepages` memory consumption. Consequently, the alert was triggered unnecessarily on the cluster for {product-title} 4.8. With this update, Linux `hugepages` have been removed from the system memory circulation, and the alert no longer unnecessarily triggers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2028854[*BZ#2028854*])
 
+* Previously, {product-title}, with OVN-Kubernetes, managed ingress access to services via ExternalIP. When upgrading from 4.8.33 to 4.8.34, access `ExternalIP` stops work with issues like "No Route to Host". With this update, administrators will now have to direct traffic from externalIPs to the cluster. For guidance, see (link:https://access.redhat.com/solutions/6954851[KCS*]) and (link:https://kubernetes.io/docs/concepts/services-networking/service/#external-ips[*Kubernetes External IPs*]) (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076662[*BZ#2076662*])
```

Comment:
I think we can say

* Previously, OVN-Kubernetes managed Ingress access to services through ExternalIPs. When upgrading from 4.8.33 to 4.8.34, accessing ExternalIPs stopped processes and returned errors like `No Route to Host`. With this update, administrators will now have to direct traffic from ExternalIPs to the cluster. For guidance, see (link:https://access.redhat.com/solutions/6954851[KCS*]) and (link:https://kubernetes.io/docs/concepts/services-networking/service/#external-ips[*Kubernetes External IPs*]) (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076662[*BZ#2076662*])